content/06.discussion.md

```diff
@@ -54,6 +54,34 @@ In assessing the upper bound on the predictive performance of a deep learning mo
+
+#### Bayesian hypernetworks
+
+Another framework for approximate Bayesian inference in deep learning is utilizing Bayesian hypernetworks [@tag:bayesian_hypernets]. A hypernetwork is a network that learns to map its input to the parameters of another network, the primary network. Both the hypernetwork and the primary network are trained together as a single model. Krueger et al introduced Bayesian hypernetworks and used an invertible generative network to map isotropic Gaussian noise to parameters of the primary network. Using an invertible hypernetwork allowed computationally cheap sampling and tractable estimation of the posterior. Bayesian hypernetworks are better at dealing with multimodal posterior distributions than other Bayesian deep learning approaches.
+
+#### Other approaches
+
+Several other techniques have been proposed for effectively estimating predictive uncertainty as uncertainty quantification for neural networks continues to be an active research area. Recently, McClure and Kriegeskorte observed that test-time sampling improved calibration of the probabilistic predictions, sampling weights led to more robust uncertainty estimates than sampling units, and spike-and-slab sampling is superior to Gaussian dropconnect and Bernoulli dropout [@tag:mcclure_bayesian]. Meanwhile, Lakshminarayanan et al proposed using deep ensembles, which are traditionally used for boosting predictive performance, on standard (non-Bayesian) neural networks to obtain well-calibrated uncertainty estimates that are comparable to those obtained by Bayesian neural networks [@tag:uncertainty_ensembles]. In cases where model uncertainty is known to be caused by a difference in training and testing distributions, domain adaptation based techniques can help mitigate the problem [@tag:domain_adapt_uncertainty].
+
+#### Future work
```

Comment:
Doesn't need a subsection. I'm going to suggest removing all the subsections. Moving bayesian hypernetworks up to a be a sentence in the bayesian NN paragraph, and maybe slightly more clearly laying out how these tools fit with the uncertainty points that you raised so clearly in the first paragraph.

Reply:
I removed all subsection titles.